app/models/item.rb

```diff
@@ -109,4 +109,9 @@ def delete_document
   def self.search_name_and_description(query)
     search(query: { multi_match: {query: query, fields: [ "name^3", "description" ]}})
   end
+
+  def all_specs_off_shelf?
+    specs_statuses = self.specs.map(&:status)
```

Comment:
this will query all specs from db, there must be more efficient way to do this test